cosmos.bunserver.ts

```diff
@@ -0,0 +1,73 @@
+import { statSync } from "fs";
+import { type ServeOptions } from "bun";
+
+const PROJECT_ROOT = import.meta.dir;
+
+async function waitForCosmosImports() {
+  const fpath = `${PROJECT_ROOT}/cosmos.imports.ts`;
+  try {
+    const cosmosImports = statSync(fpath);
+    if (!cosmosImports.isFile()) {
+      throw new Error(`
+        file doesnt exist yet
+        did you run 'bun run cosmos' ?
+      `);
+    }
+  } catch {
+    return new Promise((resolve) => {
+      setTimeout(() => resolve(waitForCosmosImports()), 1000);
+    });
+  }
+}
+await waitForCosmosImports();
+await Bun.build({
```

Comment:
i think bun accepts all of esbuild options, finish reading the bun docs on this and make a more robust build process